packages/documentation/docs/integration/deployment.md

```diff
@@ -56,7 +56,7 @@ $ helm install ...
 | `OUTGOING_PAYMENT_WORKERS`                             | `4`                                                         | number of workers processing outgoing payment requests                                                                       |
 | `OUTGOING_PAYMENT_WORKER_IDLE`                         | `200`                                                       |
 | milliseconds                                           |
-| `PAYMENT_EXPIRY_MAX_MS`                                | `86400000`                                                  | Maximum milliseconds into the future incoming payments can be set to on creation. Default: 1 day                             |
+| `INCOMING_PAYMENT_EXPIRY_MAX_MS`                       | `86400000`                                                  | Maximum milliseconds into the future incoming payments can be set to on creation. Default: 1 day                             |
```

Comment:
so srry, same thing for alphabetization 😅

Reply:
oh geeze, no need to apologize... didn't think about it after renaming :satisfied: Also made me discover that I had the default values swapped for `PAYMENT_POINTER_DEACTIVATION_PAYMENT_GRACE_PERIOD_MS` and `INCOMING_PAYMENT_EXPIRY_MAX_MS`